docs/ref-fleet-yaml.md

```diff
@@ -250,3 +250,43 @@ overrideTargets:
         env: dev
 
 ```
+
+### Helm Options
+
+#### How fleet-agent deploys the bundle
+
+These options also apply to kustomize- and manifest-style bundles.
+They control how the fleet-agent deploys the bundle. All bundles are converted into Helm charts and deployed with the Helm SDK.
+These options are often similar to the Helm CLI options for install and update.
+
+* releaseName
+* takeOwnership
+* force
+* atomic
+* disablePreProcess
+* disableDNS
+* skipSchemaValidation
+* waitForJobs
+
+#### Helm Chart Download Options
+
+These options are for Helm style bundles, they specify how to download the chart.
+
+* chart
+* repo
+* version
+
+The reference to the chart can be either:
+* a local path in the cloned Git repository, specified by `chart`.
+* a go-getter URL, specified by `chart`. This can be used to download a tarball
+  of the chart. go-getter also allows to download a chart from a Git repo.
+* a Helm repository, specified by `repo` and optionally `version`.
```

Comment:
How about describing what happens when no version is specified?
eg:
```suggestion
* a Helm repository, specified by `repo` and optionally `version` (the latest version will be fetched if none is specified).
```

Reply:
I'd need to analyze this further, but I think it's the "highest available semver tag": https://github.com/helm/helm/blob/release-3.14/pkg/downloader/chart_downloader.go#L162-L166